index.html

```diff
@@ -2,29 +2,65 @@
-    <link rel="stylesheet" type="text/css" href="guess_the_film.css">
+    <link rel="stylesheet" type="text/css" href="css/normalize.css">
+    <link rel="stylesheet" type="text/css" href="css/guess_the_film.css">
+    <script defer src="index.js"></script>
     <title>Guess the Film</title>
 </head>
 <body>
-    <main>
-        <section id="start_screen">
-            <h1>Guess the Film</h1>
-            <div class="instruction_wrapper">
-                <h2>Aim:</h2>
-                <p>Guess as many film titles from the quotes provided within 30s. Each correct answer is worth 1 point.</p>
+<main>
+    <section id="start_screen">
+        <h1>Guess the Film</h1>
+        <div class="instruction_wrapper">
+            <h2>Aim:</h2>
+            <p>Guess as many film titles from the quotes provided within 30s. Each correct answer is worth 1 point.</p>
+        </div>
+        <div class="instruction_wrapper">
+            <h2>How to play:</h2>
+            <ul>
+                <li>Press 'Start Game' to begin</li>
+                <li>Read the quote displayed, enter your guess and click 'Submit'</li>
+                <li>Pressing 'Hint' reveals 3 possible answers and the year the film was released</li>
+                <li>Click 'Next' to move to the next questions</li>
+                <li>Once a game is finished, click 'Start Again' to start a new game</li>
+            </ul>
+        </div>
+        <div>
+            <button type="button" id="start_button">Start</button>
+        </div>
+    </section>
+
+    <section id="game_screen">
+        <header id="score_time">
+            <div id="score"></div>
+            <div id="time"></div>
+        </header>
+        <h2 id="quote"></h2>
+        <article class="game_controls">
+            <div class="input_wrapper">
+                <label for="guess"></label>
+                <input name="guess" id="guess" type="text">
+                <button id="submit_button">Submit</button>
             </div>
-            <div class="instruction_wrapper">
-                <h2>How to play:</h2>
-                <ul>
-                    <li>Press 'Start Game' to begin</li>
-                    <li>Read the quote displayed, enter your guess and click 'Submit'</li>
-                    <li>Pressing 'Hint' reveals 3 possible answers and the year the film was released</li>
-                    <li>Click 'Next' to move to the next questions</li>
-                    <li>Once a game is finished, click 'Start Again' to start a new game</li>
-                </ul>
+        </article>
+        <article id="user_feedback">
+            <div id="hint"></div>
+            <div id="answer"></div>
+            <div id="correct_incorrect">
+                <div id="correct_response">You were correct!</div>
+                <div id="incorrect_response">I'm sorry you fluffed it!</div>
             </div>
-        </section>
-    </main>
+        </article>
```

Comment:
article tag isn't really appropriate for this content, it should only be used for standalone bits of content that would make sense outside the context of the rest of the application.

Reply:
Replaced with divs